packages/contractkit/src/utils/tx-params-normalizer.ts

```diff
@@ -64,7 +65,9 @@ export class TxParamsNormalizer {
   private async getNonce(address: string): Promise<number> {
     // Reference: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactioncount
     const result = await this.rpcCaller.call('eth_getTransactionCount', [address, 'pending'])
-    const nonce = parseInt(result.result.toString(), 10)
+
+    // tslint:disable-next-line: radix
+    const nonce = parseInt(result.result.toString())
```

Comment:
Why not just provide the 16 as the radix instead of disabling tslint there?
Or even better, use BigNumber which has more robust parsing then parseInt